apps/ejabberd/src/mongoose_metrics_hooks.erl

```diff
@@ -34,36 +34,17 @@
          privacy_iq_set/4,
          privacy_check_packet/6,
          user_ping_timeout/1,
-         privacy_list_push/4,
-         mam_get_prefs/4,
-         mam_set_prefs/7,
-         mam_remove_archive/3,
-         mam_lookup_messages/14,
-         mam_archive_message/9,
-         mam_flush_messages/2,
-         mam_drop_message/1,
-         mam_drop_iq/5,
-         mam_drop_messages/2,
-         mam_purge_single_message/6,
-         mam_purge_multiple_messages/9,
-         mam_muc_get_prefs/4,
-         mam_muc_set_prefs/7,
-         mam_muc_remove_archive/3,
-         mam_muc_lookup_messages/14,
-         mam_muc_archive_message/9,
-         mam_muc_flush_messages/2,
-         mam_muc_drop_message/1,
-         mam_muc_drop_iq/5,
-         mam_muc_drop_messages/2,
-         mam_muc_purge_single_message/6,
-         mam_muc_purge_multiple_messages/9
+         privacy_list_push/4
```

Comment:
I broke it into 2 modules because Elvis complained about `mongoose_metrics_hooks` being "god module".